lib/app_funcs.sh

```diff
@@ -96,13 +96,23 @@ function pre_compile_hook() {
   cd - > /dev/null
 }
 
-
 function write_profile_d_script() {
   output_section "Creating .profile.d with env vars"
   mkdir -p $build_path/.profile.d
 
   local export_line="export PATH=\$HOME/.platform_tools:\$HOME/.platform_tools/erlang/bin:\$HOME/.platform_tools/elixir/bin:\$PATH
```

Comment:
It feels like this line should be possible to change to use the functions as well, they used to but were changed in https://github.com/HashNuke/heroku-buildpack-elixir/commit/87b98c01ef435a1b9bd5acdfc433abbfbf323daf for unknown reasons.

I decided I did not want to mess with this any more than necessary.